lib/intercity_server/installer.rb

```diff
@@ -33,6 +33,14 @@ def execute
         end
       end
 
+      cli.choose do |menu|
+        menu.prompt = "Do you want to run Intercity on a custom port?\n" \
+          "This will allow you to run Intercity and your apps on the same server.\n" \
```

Comment:
Copy suggestion: "This allows you to run both the Intercity control panel and deploy your apps on this server."